libqtile/widget/backlight.py

```diff
@@ -45,10 +45,8 @@ class Backlight(base.InLoopPollText):
     If the change_command parameter is set to None, the widget will attempt to
     use the interface at /sys/class to change brightness. Depending on the
     setup, the user may need to be added to the video group to have permission
```

Comment:
We refer to `video` group here but we're not setting the group ownership to `video`.

Reply:
Thanks, need to drop this as well.